homeassistant/components/openai_conversation/config_flow.py

```diff
@@ -124,26 +127,37 @@ async def async_step_init(
     ) -> ConfigFlowResult:
         """Manage the options."""
         options: dict[str, Any] | MappingProxyType[str, Any] = self.config_entry.options
+        errors: dict[str, str] = {}
 
         if user_input is not None:
             if user_input[CONF_RECOMMENDED] == self.last_rendered_recommended:
                 if user_input[CONF_LLM_HASS_API] == "none":
                     user_input.pop(CONF_LLM_HASS_API)
-                return self.async_create_entry(title="", data=user_input)
-
-            # Re-render the options again, now with the recommended options shown/hidden
-            self.last_rendered_recommended = user_input[CONF_RECOMMENDED]
-
-            options = {
-                CONF_RECOMMENDED: user_input[CONF_RECOMMENDED],
-                CONF_PROMPT: user_input[CONF_PROMPT],
-                CONF_LLM_HASS_API: user_input[CONF_LLM_HASS_API],
-            }
+                if (
+                    user_input.get(CONF_LLM_HASS_API)
```

Comment:
No need to use `get`, it will always be present. The previous check was like that too.

Reply:
We could remove it on the previous line (`user_input.pop(CONF_LLM_HASS_API)`)

Reply:
Done